### ISSUE TITLE
error after upload artboard

### ISSUE THREAD

**Issue by mekrukov** [avocode/avocode#1554]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.7`
Avocode UID: 28316
```
Error: ENOENT: no such file or directory, stat '/var/folders/jz/35bfl6ld1hd0jmcs0kqmwjkc0000gp/T/com.madebysource.avocode/avocode-sync-1501065551523/3A241774-102B-4969-BA33-EE5BEEE6C3E8/1B823EA2-B4AB-4C2F-A396-D2117DFB28DC.png'
```

**Comment by jakubzitny** [comment]
Hi @mekrukov, thanks for reporting this. This is a duplicate of #1551, we're having problems with updating the Sketch plugin in the latest Avocode and Sketch. Can you please try to download [this version](https://upx.cz/iYd), double click to update it in Sketch and try again?

**Comment by jakubzitny** [comment]
Hi, we've fixed this problem in latest Avocode `2.14.8`, please update the app and let us know if you have other problems or issues :wink:
